### DISCUSSION TITLE
Use of SSL certificates

### DISCUSSION THREAD

**Discussion by daniel-lara-ec** [sfu-db/connector-x#337] [Q&A]
Hi, 

I want to connect to my database, but I don't know how to use ca_cert. Could anyone help me? I didn't find anything in documentation.

**Comment by wangxiaoying** [comment]
Hi @daniel-lara-ec , can you see whether this works? https://github.com/sfu-db/connector-x/issues/222#issuecomment-1084926988

**Reply by daniel-lara-ec** [reply]
Hi, I tried the solution but give this error:
![image](https://user-images.githubusercontent.com/53488713/188044309-9c219dd5-a23e-4e31-9d52-e67c71d2f0bd.png)

**Reply by wangxiaoying** [reply]
Hi @daniel-lara-ec , it seems like you are using mysql according to your connection string. Unfortunately, we do not support SSL certificates for mysql right now. I've opened this [issue](https://github.com/sfu-db/connector-x/issues/347) to keep this in track.

**Reply by daniel-lara-ec** [reply]
Hi @wangxiaoying, yes I use mysql. Thanks for your help!
